src/context/FilterProjectsContext.jsx

```diff
@@ -489,6 +489,18 @@ export const FilterProjectsProvider = ({ children }) => {
     setShowYourData(checked)
   }
 
+  const projectCount = useMemo(() => {
```

Comment:
Any reasons for `useMemo` here?

Reply:
I thought I needed it but looks like it's not necessary